unify/utils/evaluations.py

```diff
@@ -338,6 +341,7 @@ def log(
         kwargs = {
             k + "/" + version[k] if k in version else k: v for k, v in kwargs.items()
         }
+    kwargs = {**kwargs, **current_global_active_log_kwargs.get()}
```

Comment:
Let's make this something like: `kwargs = {**kwargs, **{k: v[0] for k, v in current_global_active_log_kwargs.get() if not v[1]}}`

Reply:
then on a new line add something like: `current_global_active_log_kwargs = {k: (v[0], True) for k, v in current_global_active_log_kwargs.get()}`

Reply:
Lets then also add this same logic to `add_log_entries`